src/graph.py

```diff
@@ -116,6 +116,66 @@ def breadth_traversal(self, start):
                     if edge not in checked]
         return checked
 
+    def heuristic(self, start):
+        """Run BFS to set a heuristic for A* algorithm on a simple weighted graph."""
+        node_list = [start]
+        checked, weights = [], []
+        while node_list:
+            vertex = node_list.pop(0)
+            if vertex not in checked:
+                checked.append(vertex)
+                for key, weight in self.graph[vertex].items():
+                    if self.graph[vertex][key] not in checked:
+                        weights.append(weight)
+                        node_list.append(key)
+        print("weights: " + str(weights))
```

Comment:
Nice for debugging, but you should probably remove this when you are sure it is working correctly